src/browser/commands/types.ts

```diff
@@ -1,87 +1,6 @@
-import { AssertViewOptsConfig } from "../../config/types";
+import { AssertViewOpts } from "../../config/types";
 import { ChainablePromiseElement } from "webdriverio";
 
-export interface AssertViewOpts extends Partial<AssertViewOptsConfig> {
```

Comment:
there is no point in creating it separately (`AssertViewOpts` and `AssertViewOptsConfig`), because they have the same fields